static/app/views/alerts/rules/issue/details/ruleDetails.spec.jsx

```diff
@@ -18,12 +18,17 @@ describe('AlertRuleDetails', () => {
     lastTriggered: moment().subtract(2, 'day').format(),
   });
 
-  const createWrapper = (props = {}) => {
+  const createWrapper = (props = {}, queryParameter) => {
     const params = {
       orgId: organization.slug,
       projectId: project.slug,
       ruleId: rule.id,
     };
+
+    if (queryParameter) {
+      context.router.location.query = {mute: '1'};
```

Comment:
You don't want to mutate this variable, otherwise any tests you write after one with `queryParameter=true` will have query set this way. You probably want to spread this into a new object instead?